docker-compose-development.yml

```diff
@@ -337,4 +337,4 @@ services:
       sdkEnv: local
       envSdkUrl: http://localhost:9050
       envBackendUrl: http://localhost:8080
-      envLoggingUrl: http://localhost:8207
+      envLoggingUrl: http://localhost:8207
```

Comment:
revert this change

Reply:
I tried to revert this, I thought it was about line endings and tried a couple of things but didn't work